modules/apps/object/object-test/src/testFunctional/tests/macros/ObjectAdmin.macro

```diff
@@ -10,6 +10,27 @@ definition {
 		Click(locator1 = "ObjectAdmin#SAVE_CUSTOM_OBJECT");
 	}
 
+	macro deleteAllCustomObjects {
+		ObjectAdmin.openObjectAdmin();
+
+		if (IsElementNotPresent(locator1 = "ObjectAdmin#VIEW_CUSTOM_OBJECT")) {
+			ObjectAdmin.notViewSystemObject();
+		}
+		else {
```

Comment:
You can remove this `if` and `else`. Let's keep only the `while`.

Reply:
@rodrigocunhaa Was removed.